semeio/workflows/localisation/local_script_lib.py

```diff
@@ -98,10 +99,8 @@ def get_param_from_ert(ens_config):
         impl_type = node.getImplementationType()
         my_param = Parameter(key, impl_type)
         new_params.append(my_param)
-        if impl_type == ErtImplType.GEN_KW:
-            # Node contains scalar parameters defined by GEN_KW
-            kw_config_model = node.getKeywordModelConfig()
-            my_param.parameters = kw_config_model.getKeyWords().strings
+        if isinstance(node, GenKwConfig):
```

Comment:
Why this substitution is not needed in line 675?

Reply:
This is because Gen_data is of type EnkfConfigNode, and not GenKwConfig.
They were both EnkfConfigNodes at some point though, but the refactoring separated GEN_KW into its own entity.

```
      elif impl_type == ErtImplType.GEN_DATA:
```